.github/workflows/golangci-lint.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         with:
           go-version: ${{ steps.vars.outputs.go_version }}
       - name: golangci-lint-${{matrix.working-directory}}
-        uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
+        uses: golangci/golangci-lint-action@3cfe3a4abbb849e10058ce4af15d205b6da42804 # v4.0.0
```

Comment:
This is funny:

```
commit 3cfe3a4abbb849e10058ce4af15d205b6da42804 (tag: v4.0.0, tag: v3.7.1)
```

Same commit is BOTH v4.0.0 and v3.7.1.

Reply:
Added commit on top to fix the comment tag, so next round it doesn't go and bump it to 4.x further.